_includes/android/user-interface.md

```diff
@@ -178,143 +178,145 @@ You can override any layout resources by having files with the same name as thos
-
-```java
-@Override
-public View getItemView(ParseObject object, View v, ViewGroup parent) {
-  if (v == null) {
-    v = View.inflate(getContext(), R.layout.adapter_item, null);
-  }
-  v.setBackgroundColor(object.getInt("color"));
-  return v;
-}
-
-@Override
-public View getNextPageView(View v, ViewGroup parent) {
-  if (v == null) {
-    // R.layout.adapter_next_page contains an ImageView with a custom graphic
-    // and a TextView.
-    v = View.inflate(getContext(), R.layout.adapter_next_page, null);
-  }
-  TextView textView = (TextView) v.findViewById(R.id.nextPageTextViewId);
-  textView.setText("Loaded " + getCount() + " rows. Get more!");
-  return v;
-}
-
-```
-
-## Loading Remote Images in Rows
-
-`ParseQueryAdapter` makes it simple to display remote images. By calling `setImageKey(String)`, you can pass in a key name on your `ParseObject` which should contain a `ParseFile` containing an image to be fetched from Parse and loaded into the `ParseImageView` of the corresponding row.
-
-The image will download asynchronously, and the appropriate `ParseImageView` will be updated in the background. As the user scrolls and rows are recycled by the adapter, images will be fetched as rows become visible and assigned `ParseObject`s.
-
-You can define a placeholder image to be used when the image fetch has not yet completed. Call `setPlaceholder(Drawable)` on your `ParseQueryAdapter` to use the specified `Drawable` as a fallback image.
-
-## Lifecycle Methods
-
-We expose two hooks in the data lifecycle of the Adapter for you to execute custom logic &mdash; right before we query Parse for your data and right after the fetched objects have been loaded from the query. These methods are particularly useful for toggling some loading UI.
-
-An `OnQueryLoadListener` can be set via `setOnQueryLoadListener(OnQueryLoadListener)`, which provides `onLoading()` and `onLoaded(List<ParseObject>, Exception)` methods for implementation.
-
-## Pagination
-
-Pagination ensures that the table only gets one page of objects at a time. You can set the number of objects are in a page by calling `setObjectsPerPage(int)`.
-
-The query is automatically altered to apply pagination, and a pagination row appears at the bottom of the `AdapterView` to allow users to load the next page.
-
-Pagination is turned on by default. To turn it off, call `setPaginationEnabled(false)`. With pagination turned off, the `ParseQueryAdapter` will use the default `ParseQuery` limit of 100 items.
-
-## Auto-loading of Data
-
-When the `AdapterView` that your `ParseQueryAdapter` is set on is attached to a window, the `ParseQueryAdapter`'s `loadObjects()` method is automatically called, triggering the fetching of the first page of results. To disable this behavior (perhaps to delay the fetching of data, or run some custom logic ahead of time), just call `setAutoload(false)` and call `loadObjects()` manually if autoload is disabled.
+## ParseQueryAdapter (Deprecated 0.0.4)
```

Comment:
This is odd that all the lines are changed.. What machine / OS are you on?

Reply:
Hmm, that is odd. macOS High Sierra.

Reply:
There were a few lines added, like the deprecation notice. But yes, the rest did stay the same as before.